### DISCUSSION TITLE
How to run the inference of the trained MONAI Deepgrow model?

### DISCUSSION THREAD

**Discussion by klchtsk** [Project-MONAI/MONAILabel#1760] [Q&A]
I have trained a MONAI Deepgrow model. How to correctly run the inference of the model to check its performance? Do I need to convert it to segmentation model?

![image](https://github.com/user-attachments/assets/f974eb68-80ea-4860-b012-11f2aa950401)

**Comment by diazandr3s** [comment]
Hi @klchtsk,

You should be able to run inference using the MONAI Label Slicer module. The infer button is just below the bars you've shared.
BTW, have you tried the SAM2 model available in the latest release of MONAI Label?

Hope this helps,
